### DISCUSSION TITLE
How to color the limits of integration

### DISCUSSION THREAD

**Discussion by Rookiefrog** [ManimCommunity/manim#2984] [Q&A] [answered]
Is it possible to color the limits of an integration without changing the color of the integration sign?
Something like the following code is close, but it still colors the integration sign.
`obj = MathTex(r"\int\limits_{a}^{b}","f(x)","dx")`
`obj.set_color_by_tex_to_color_map({"a":RED,"b":RED,"f(x)":GREEN,"dx":BLUE})`
![image](https://user-images.githubusercontent.com/66852299/194731748-c6c31df4-a49d-47bb-9f24-140a5360f50a.png)

**Comment by SuperUser-Vermeylen** [comment, marked as answer]
I think the issue is that your obj has the limits of integration as a single string.
Break that into multiple strings in the obj and then you should have more control.

Check this [out](https://docs.manim.community/en/stable/reference/manim.mobject.text.tex_mobject.MathTex.html).
I think you just need to get your curly braces set up correctly.

**Reply by Rookiefrog** [reply]
Thanks
